bin/prepare-release.sh

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env bash
```

Comment:
I suggest that we use [ShellCheck](https://www.shellcheck.net/) as a linter for shell scripts.

Reply:
I'll note that you only have one minor problem with a comment, this is really great as SC is super picky :)

Reply:
Heh :) And I think that's even a false positive!

The error is:
 
```
Line 64:
    echo 'To proceed despite this and include the uncommited changes, set the `ALLOW_DIRTY` environment variable.'
         ^-- SC2016: Expressions don't expand in single quotes, use double quotes for that.
```

But we don't want that to expand, we want it to print `ALLOW_DIRTY`

Reply:
heh